internal/helpers.go

```diff
@@ -65,6 +65,11 @@ func GenerateForeignkeyId() string {
 func GenerateIndexesId() string {
 	return GenerateId("i")
 }
+
+func GenerateCheckConstrainstId() string {
+	return GenerateId("ck")
```

Comment:
nit: ck is non obvious - can we use cc or something else here ?

Reply:
this has been incorporated to the code.